database/db.go

```diff
@@ -15,19 +15,24 @@ var (
 
 // DBConfig wraps config for connecting to a database.
 type DBConfig struct {
-	Address  string
-	User     string
-	Database string
-	Password string
+	Address       string
+	User          string
+	Database      string
+	Password      string
+	Logger        *log.Logger
+	EnableLogging bool
 }
 
 // DB wraps a client for a database.
 type DB struct {
-	Client *pg.DB
+	Client      *pg.DB
+	Logger      *log.Logger
```

Comment:
I think this should be embedded into this struct and not a public field(I can't think of a use case other code would have for needing access to the database's logger), with the additional upside that methods on this struct can make logging statements via `db.Println()` versus `db.Logger.Println` 
https://golang.org/doc/effective_go.html#embedding
https://travix.io/type-embedding-in-go-ba40dd4264df

Reply:
Note though that a service logger satisfies the logger interface so can be passed as the logger value here, and that the underlying Postgres client only accepts a logger interface so won't be able to take advantage of any additional methods we have(though our database package can)

Reply:
As far as I know `log.Logger` is still a concrete type, not an interface :(

https://github.com/golang/go/issues/13182